presto-pinot-toolkit/src/test/java/com/facebook/presto/pinot/query/TestPinotQueryGeneratorSql.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.pinot.query;
+
+import com.facebook.presto.testing.assertions.Assert;
+import org.testng.annotations.Test;
+
+public class TestPinotQueryGeneratorSql
+        extends TestPinotQueryGenerator
```

Comment:
Is order by aggregation limit tested ? That's another difference b/w the SQL and the PQL end point. For example, what would be the pinot sql corresponding to this presto sql: "select A, B, Count(1), Sum(C) from Table group by B, A order by SUM(C) ASC limit 3" 

Another example: "select A, B, Sum(C) from Table group by B, A order by SUM(C) ASC limit 3"

Reply:
Will add.

Reply:
Added.